docker/requirements.txt

```diff
@@ -4,7 +4,9 @@ black
 cmakelint
 cpplint
 deprecated
-docformatter
+# Pin due to formatting conflict with black.
+# https://github.com/PyCQA/docformatter/issues/94
+docformatter==1.5.1
```

Comment:
I'm not sure if this is still required, but it's in the main `requirements.txt` file in the directory above, so I added it here too.